msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -0,0 +1,381 @@
+
+    This step-by-step guide shows a feature where users can broadcast a message to all members of the organization using activity feed notification.
+
+    **Interactions of graph activity feed broadcast**
+
+    * Broadcasts message for all the members.
+    * Displays activity feed notification.
+    * Broadcasts details available for members. 
+
+    You'll see the following output:
+
+      :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/outputready.png" alt-text="Output of feed broad cast" border="true":::
+
+- title: Prerequisites
+  durationInMinutes: 1
+  content: |
+    To set up your development environment, install the following:  
+
+    * [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+    * Latest version of Visual Studio
+    * [ngrok](https://ngrok.com/download) latest version (only for devbox testing) or any equivalent tunneling solution
+    
+      > [!NOTE]
+      > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
+    
+    * [Teams](https://teams.microsoft.com/) with valid account
+
+- title: Set up local environment
+  durationInMinutes: 1
+  content: |
+   Clone `Microsoft-Teams-Samples` repository to your local GitHub:  
+
+    1. Open [Microsoft Teams Samples](https://github.com/OfficeDev/Microsoft-Teams-Samples).
+    1. Select **Code**.
+    1. From the dropdown menu, select **Open with GitHub Desktop**.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/clonerepository.png" alt-text="Clone repository" border="true":::
+
+    1. Select **Clone**. 
+
+- title: Register your Teams auth single sign-on (SSO) with Azure Active Directory (Azure AD)
+  durationInMinutes: 1
+  content: |
+    **To register your app through the Azure AD portal**  
+
+    1. Go to the [Azure portal] (https://portal.azure.com/).
+
+    1. Select **Azure Active Directory**.
+
+    1. In the left navigation panel, select **App Registrations**.
+
+    1. Select **New registration**.
+
+       :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/new-app-registration.png" alt-text="New registration" border="true":::
+
+    1. On the **Register an application** page, set the value as follows:
+
+       * Set **Name** to your app name.
+
+       * Set **Supported account types** to any account type.
+
+       * Leave the **Redirect URI** empty.
+
+       * Select **Register*.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/register.png" alt-text="New registration" border="true":::
+
+    1. From the app **Overview** page, copy the values of the **Application (client) ID** and **Directory (tenant) ID** and save the values for later use.
```

Comment:
Update the steps to get the client ID and tenant ID, the step seems abrupt or else. Provide connecting sentence/ steps or paragraph before the step: From the app Overview page, copy the values of the Application (client) ID and Directory (tenant) ID and save the values for later use.